oak_restricted_kernel_runtime/src/syscall.rs

```diff
@@ -0,0 +1,137 @@
+//
+// Copyright 2022 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+use crate::{raw_syscall::Syscall, syscall};
+use core::ffi::{c_int, c_size_t, c_ssize_t, c_void};
+use strum::{Display, FromRepr};
+
+#[allow(clippy::upper_case_acronyms)]
+#[derive(Debug, Display, Eq, FromRepr, PartialEq)]
+#[repr(isize)]
+#[non_exhaustive]
+pub enum Errno {
+    /// Input/output error
+    EIO = -5,
```

Comment:
Are these meant to match standard linux errors? If so, please add a comment with references, otherwise if someone needs to add more in the future they will need to figure it out (or they will pick an incorrect value).

Reply:
Added comments.